site/content/docs/5.1/components/orange-footer.md

```diff
@@ -7,31 +7,455 @@ aliases: "/docs/5.1/components/orange-footer/"
+      <li><a href="#" class="btn btn-icon btn-social btn-mail btn-inverse"><span class="visually-hidden">Mail</span></a></li>
+    </ul>
+  </div>
+
+  <div class="border-bottom border-1 border-dark"></div>
+
+  <!-- Navigation -->
+  <div class="navbar-dark footer-nav">
+    <div class="container d-none d-md-flex">
+      <div class="col-3 me-1">
+        <h3>Category</h3>
+        <ul class="navbar-nav">
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+        </ul>
+      </div>
+      <div class="col-3 me-1">
+        <h3>Category</h3>
+        <ul class="navbar-nav">
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+        </ul>
+      </div>
+      <div class="col-3 me-1">
+        <h3>Category</h3>
+        <ul class="navbar-nav">
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+        </ul>
+      </div>
+      <div class="col-3 me-1">
+        <h3>Category</h3>
+        <ul class="navbar-nav">
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+          <li><a class="nav-link" href="#">Subcategory</a></li>
+        </ul>
+      </div>
+    </div>
+    <div class="d-md-none accordion" id="accordionExampleSmall">
```

Comment:
maybe the use of accordion-sm could reduce the number of additionnal css needed in footer.scss?

Reply:
The way they are developped, it wouldn't change anything here : Same padding everywhere, only the line-height changes.